spotbugs-plugin/src/test/java/com/linkedin/avroutil1/spotbugs/GeneratedTestClass.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2022 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avroutil1.spotbugs;
+
+import org.apache.avro.Schema;
+
+
+public class GeneratedTestClass extends org.apache.avro.specific.SpecificRecordBase
+    implements org.apache.avro.specific.SpecificRecord {
+  @Override
+  public Schema getSchema() {
+    return null;
+  }
+
+  @Override
+  public Object get(int field) {
+    return null;
+  }
+
+  @Override
+  public void put(int field, Object value) {
+
```

Comment:
you should add some code here that would normally trigger a violation, like Schema.toString() or something, no?

Reply:
Added a method which violates toString schema serialization rule.